lib/travis/api/v3/queries/build.rb

```diff
@@ -37,10 +37,8 @@ def restart(user)
     end
 
     def priority(user)
```

Comment:
Do we really need user here?

Reply:
As of now user is not needed here. We have removed user from here.